lib/crowbar/init/helpers.rb

```diff
@@ -254,6 +254,18 @@ def update_config_db
         )
       end
 
+      def ensure_admin_node_exist
+        # temporary workaround to enable gating for crowbar-core#920
+        return true unless File.exist?("/opt/dell/crowbar_framework/app/models/chef_node.rb")
```

Comment:
It's possible to re-trigger the gating job from crowbar-core#920 with this patch also. Do you want to get this merged now anyway (in which case, I guess we could keep this condition), or at the same time as crowbar-core#920 (in which case we don't need the condition)?